packages/core/src/audio/AudioManager.ts

```
@@ -3,48 +3,114 @@
+        return null;
+      }
     }
     return context;
   }
 
-  static getGainNode(): GainNode {
+  static getGainNode(): GainNode | null {
+    if (!AudioManager._isAudioAvailable) {
+      return null;
+    }
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)